packages/web-runtime/src/index.ts

```diff
@@ -41,16 +41,36 @@ import PortalVue, { createWormhole } from 'portal-vue'
 
 import Avatar from './components/Avatar.vue'
 import focusMixin from './mixins/focusMixin'
+import { ArchiverService } from 'web-pkg/src/services/archiver'
+import { get } from 'lodash-es'
 
 export const bootstrapApp = async (configurationPath: string): Promise<void> => {
   const app = createApp(pages.success)
 
+  app.provide('$router', router)
+
   const runtimeConfiguration = await announceConfiguration(configurationPath)
   startSentry(runtimeConfiguration, app)
 
   const store = await announceStore({ runtimeConfiguration })
+  app.provide('$store', store)
   app.use(abilitiesPlugin, createMongoAbility([]), { useGlobalProperties: true })
 
+  // TODO: move to announceArchiverService function
```

Comment:
I will be juggling around quite a bit with init order, so I would like to tackle this in a follow up